src/main/scala/scalajsGames/games/Snake.scala

```diff
@@ -12,6 +12,7 @@ case object Empty extends Spot
 case class Snake(bounds: Point, resetGame: () => Unit) extends Game {
   var frameCount = 0
   var length = 10
+  var score = 0
```

Comment:
isn't necessary to reset the value when you are dead ?

Reply:
It isn't necessary because after the game ends the score resets by the resetGame() function